docs/man/man1/oc-adm-migrate-scc.1

```diff
@@ -0,0 +1,3 @@
+This file is autogenerated, but we've stopped checking such files into the
+repository to reduce the need for rebases. Please run hack/generate-docs.sh to
+populate this file.
```

Comment:
Should this file not be checked  in ?

Reply:
As far I understand, yes, perhaps @juanvallejo knows more about it.